include/ublksrv.h

```diff
@@ -563,7 +563,13 @@ extern void ublksrv_ctrl_prep_recovery(struct ublksrv_ctrl_dev *dev,
 		const char *recovery_jbuf);
 
 /**
- * Return device's json buffer
+ *
+ * Get json buffer from device
+ */
+extern const char *ublksrv_ctrl_get_jbuf(const struct ublksrv_ctrl_dev *cdev);
```

Comment:
If we deal with any json read/write via `ublksrv_ctrl_dev`, the json buffer becomes one private structure, and is supposed to not expose to user.